UnitTests/GitUI.Tests/ThreadHelperTests.cs

```diff
@@ -27,6 +27,7 @@ private static async Task ThrowExceptionAsync(Exception ex)
         }
 
         [Test]
+        [Ignore("Hangs")]
         public async Task FileAndForgetReportsThreadException()
         {
             using (var helper = new ThreadExceptionHelper())
```

Comment:
@sharwell this and below tests hang in `await AsyncTestHelper.JoinPendingOperationsAsync(AsyncTestHelper.UnexpectedTimeout);`.